backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -1006,7 +1006,8 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
       // the state names.
       // This logging and metadata publishing assumes that StandardAsyncRunState subtypes `toString` nicely to state names.
       val prevStatusName = previousState.map(_.toString).getOrElse("-")
-      jobLogger.info(s"Status change from $prevStatusName to $state")
+      if (prevStatusName == state.toString) jobLogger.debug(s"Status change from $prevStatusName to $state")
```

Comment:
I don't think this change is needed, the `if (!(previousState exists statusEquivalentTo(state)))` on line 1004 already makes sure we log this at the right time

Reply:
Was not aware this was changed. All I did is fixing the merge conflicts ;)
Did remove this change again, seems to work fine indeed.